3-platform/order/spec/src/main/java/com/_6core/platform/orderspec/rest/v1/OrderGetApi.java

```diff
@@ -0,0 +1,63 @@
+package com._6core.platform.orderspec.rest.v1;
+
+import com._6core.platform.orderspec.rest.v1.dto.OrderDetailsResponse;
+import com._6core.platform.orderspec.rest.v1.dto.OrderItemResponse;
+import com._6core.platform.orderspec.rest.v1.dto.OrderResponse;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import java.math.BigInteger;
+import java.util.Set;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import reactor.core.publisher.Flux;
+
+@Tag(name = "Order Receive Specification", description = "REST reactive specification for interaction with Order service")
+public interface OrderGetApi {
+
+    /**
+     * Searching orders history by single user GET /orders
+     * 
+     * @return OK (status code 200)
+     */
+    @GetMapping(value = "/orders", produces = "application/json")
+    @Operation(summary = "Get orders history", description = "Get all orders with all statuses by a single user")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "OK", content = {
+                    @Content(mediaType = "application/json", schema = @Schema(implementation = OrderResponse.class)) }),
+            @ApiResponse(responseCode = "401", description = "Authorization information is missing or invalid"),
+            @ApiResponse(responseCode = "500", description = "Unexpected error") })
+    default Flux<ResponseEntity<OrderResponse>> getOrdersHistory() {
```

Comment:
OrderResponse... I didn't get from the name about what this class. 
```suggestion
    default Flux<ResponseEntity<OrderResponse>> getOrdersHistory() {
```

Reply:
Is OrderGeneralResponse would be better?